ibtk/include/ibtk/FEDataManager.h

```diff
@@ -239,11 +239,6 @@ class FEDataManager : public SAMRAI::tbox::Serializable, public SAMRAI::mesh::St
      */
     struct WorkloadSpec
     {
-        /// Whether or not to clear the workload index arrays at the start of
-        /// the calculation. This should be true unless some other object is
-        /// also adding values into the workload estimate.
-        bool clear_estimate = true;
-
```

Comment:
Clearing is now done by the parent integrator.

Reply:
The sheer ad-hocness of that approach in FEDataManager (which, I admit, was all my work) was the main motivation to bite the bullet and redo everything here :)